platform/core/src/services/DisplaySetService/DisplaySetService.js

```diff
@@ -194,6 +194,11 @@ export default class DisplaySetService {
     }
   };
 
+  onModeExit() {
```

Comment:
The display set cache was growing without bound because it wasn't getting cleared.

Reply:
what do you mean? so it was re-creating the same displaySets? we should not create instead. This enforcing of the DS clearance on mode is not a universal thing that every one wants for performance reasons.

Reply:
You have to clear the DS values on the end of the mode because there is no guarantee that the same DSes would be created, and there isnt' any good way to check it nor to ensure that the right callbacks are made as the DSes get created.  Creating the DS is relatively cheap once one has the DICOM metadata, even for large numbers of frames/instances, so I don't think that is an issue.